### PR TITLE
add locator func to frrcfgd

### PR DESCRIPTION
**Pull request by Yubin-Li**
##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by eddieruan-alibaba** [comment]
Has cherry-picked in Phoenixwing 

https://github.com/eddieruan-alibaba/sonic-buildimage/commit/ae030c3424242ef1ef36f03afbc24b1c769ca69d

File: src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py [hunk 1]
```diff
@@ -401,6 +402,47 @@ def get_command_cmn(daemon, cmd_str, op, st_idx, vals, bool_values):
         cmd_args.append(CommandArgument(daemon, cmd_enable, vals[idx]))
     return [cmd_str.format(*cmd_args, no = CommandArgument(daemon, cmd_enable))]
 
+def hdl_srv6_locator(daemon, cmd_str, op, st_idx, vals, bool_values):
```
Comment (lguohan) [inline src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py:405]: no unit tests at all?

Comment (eddieruan-alibaba) [reply src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py:405]: https://github.com/eddieruan-alibaba/sonic-mgmt/blob/eruan-testcasemaster/ansible/vars/configdb_jsons/7nodes_cisco/PE1.json#L8

in https://github.com/sonic-net/sonic-mgmt/pull/13785

Result could be found at https://aisportal.oss-cn-hangzhou.aliyuncs.com/phoenixwing/log/basesanity/basesanity_242_log.tar

Test results sneak view. 

srv6/test_srv6_basic_sanity.py::test_interface_on_each_node PASSED       [ 14%]
srv6/test_srv6_basic_sanity.py::test_check_bgp_neighbors PASSED          [ 28%]
srv6/test_srv6_basic_sanity.py::test_check_routes PASSED                 [ 42%]
srv6/test_srv6_basic_sanity.py::test_traffic_check PASSED                [ 57%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_local_link_fail_case PASSED [ 71%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_remote_igp_fail_case PASSED [ 85%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_remote_bgp_fail_case PASSED [100%]

**Review by venkatmahalingam — APPROVED** [review]

**Review by eddieruan-alibaba — APPROVED** [review]

**Comment by eddieruan-alibaba** [comment]
@lguohan @kperumalbfn  Need help to review this PR.

**Pull request merged**